packages/firestore/src/util/error.ts

```diff
@@ -116,6 +116,12 @@ export const Code = {
    */
   ALREADY_EXISTS: 'already-exists' as FirestoreErrorCode,
 
+  /**
+   * Calling initializeFirestore() after a Firestore instance has already been created for the app
+   * It can happen when initializeFirestore() is called more than once or after calling getFirestore().
+   */
+  ALREADY_INITIALIZED: 'already-initialized' as FirestoreErrorCode,
```

Comment:
These are GRPC error codes. They cannot be modified. You should use `FAILED_PRECONDITION`.

Reply:
done